low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -325,21 +324,35 @@ def sspl_bundle_cleanup():
     if os.path.exists(sspl_bundle_tmp):
         shutil.rmtree(sspl_bundle_tmp)
 
+def validate_sb_request():
+    """Checking previous process pid/request is running or not."""
+    valid_request = True
+    sb_pid = None
+    if os.path.isfile(onGoingRequestPath):
+        with open(onGoingRequestPath,'r') as f:
+            sb_pid = f.read()
+        if sb_pid and psutil.pid_exists(int(sb_pid)):
+            valid_request = False
```

Comment:
this valid_request  var name is confusing, can we rename to 
valid_request  -> allow_new_request

Reply:
Done